### DISCUSSION TITLE
[Question][Hw5] Gradescope autograder failed to execute correctly

### DISCUSSION THREAD

**Discussion by 702fbtngus** [prosyslab-classroom/cs524-program-analysis#339] [Q&A] [answered]
Name: Suhyeon Ryu

Hello. I just submitted my ThriLLVM Analyzer to Gradescope, but it says "The autograder failed to execute correctly." How can I get help with this? I think I already passed the public test and coverage requirement with my VM.
This is the link to my Gradescope submission. https://www.gradescope.com/courses/553974/assignments/2976276/submissions/211548394?view=results
Thank you.
![image](https://github.com/prosyslab-classroom/is593-2023-fall-hw5-702fbtngus/assets/79245586/0dc9f06d-4094-47e9-b2dd-3d062a73f055)
![image](https://github.com/prosyslab-classroom/is593-2023-fall-hw5-702fbtngus/assets/79245586/9590c228-23d2-4599-b2a2-dcc65604242c)

**Comment by yeonhee-ryou** [comment, marked as answer]
It seems that you have changed the code template associated with autograder. Please check your code again.

**Comment by 702fbtngus** [comment]
I accidentally commented out the line printing # Memory operations. It was fixed by uncommenting the line. Thank you!
